src/main/java/com/revature/services/SwaggerSummaryService.java

```diff
@@ -0,0 +1,79 @@
+package com.revature.services;
+
+import java.util.Map;
+import javax.persistence.EntityManager;
+
+import com.revature.models.SwaggerSummary;
+import com.revature.models.SwaggerUploadResponse;
+import com.revature.repositories.SwaggerSummaryRepository;
+import com.revature.templates.LoadTestConfig;
+import io.swagger.models.HttpMethod;
+import io.swagger.models.Operation;
+import io.swagger.models.Path;
+import io.swagger.models.Swagger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class SwaggerSummaryService {
+
+    @Autowired
+    private EntityManager entityManager;
+
+    @Autowired
+    private SwaggerSummaryRepository repository;
+
+    public SwaggerSummary insert() {
+
+        SwaggerSummary s = new SwaggerSummary();
+
+        return repository.save(s);
+
+    }
+
+    public void update(SwaggerSummary s) {
```

Comment:
Might want some confirmation as a return type so we can let the user know if there was a problem updating.